news/_posts/2016-06-29-release-notes-2.12.0-M5.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: news
+post-type: announcement
+permalink: /news/2.12.0-M5
+title: "Scala 2.12.0-M5 is now available!"
+---
+We are happy to announce the availability of Scala 2.12.0-M5!
+
+Notable changes in M5
+([see here](https://github.com/scala/scala/pulls?utf8=%E2%9C%93&q=milestone%3A2.12.0-M5%20label%3Arelease-notes)
+for a more extensive list):
+  - [#5251](https://github.com/scala/scala/pull/5251): concrete trait methods are compiled to a
```

Comment:
It would be really great if scala-lang could use the exact same markdown formatter as github. I know we've made some progress, but I forgot where we left it. OTOH, maybe we can just use the github release notes page for these stories